_data/blog_posts.yml

```diff
@@ -1,3 +1,12 @@
+- title: "A talk on the new AWS Application Load Balancer, updates to ECS, and Kinesis Analytics"
+  url: "https://blog.gruntwork.io/a-talk-on-the-new-aws-application-load-balancer-updates-to-ecs-and-kinesis-analytics-abb599cb3cb8#.vhbek5ce2"
+  thumbnail_url: "https://i.imgur.com/KkniqPD.png"
```

Comment:
Nice find! Unfortunately, I don't think they support SlideShare's quite right. Although I get a message box confirming the SlideShare is now the featured image the green border doesn't turn red and the thumbnail never actually adjusts.

It doesn't look like it's possible to set a thumbnail of an image not in the story?